src/ring_theory/jacobson.lean

```diff
@@ -122,6 +121,26 @@ lemma is_jacobson_iso (e : R ≃+* S) : is_jacobson R ↔ is_jacobson S :=
 ⟨λ h, @is_jacobson_of_surjective _ _ _ _ h ⟨(e : R →+* S), e.surjective⟩,
   λ h, @is_jacobson_of_surjective _ _ _ _ h ⟨(e.symm : S →+* R), e.symm.surjective⟩⟩
 
+lemma is_jacobson_of_is_integral [algebra R S] (hRS : algebra.is_integral R S)
```

Comment:
If it turns out that the refactor I suggested above means that you need to make a lot of small changes in various files it might be more hygienic from a library-development point of view to put this lemma in a small follow-up PR. (Seperate refactors from new content.)

Reply:
I removed these since the refactor got fairly large